src/components/styles/displayBook.css

```diff
@@ -0,0 +1,132 @@
+.displayBook {
+  display: flex;
+  justify-content: space-between;
+  width: 94%;
+  height: 8.5em;
+  margin: 0 0.063em 0.938em 0;
+  padding: 2em 2.5em;
+  border-radius: 4px;
+  border: solid 1px #fff;
+  background-color: #fff;
+}
+
+.book-details {
+  width: 50%;
+}
+
+.progress {
+  width: 50%;
+  padding: 0 5em 0 10em;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.category-name {
+  color: rgb(12, 12, 12, 0.4);
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1em;
+}
+
+.book-title {
+  font-family: 'Montserrat', sans-serif;
+  color: black;
+  font-size: 1.5em;
+}
+
+.book-author {
+  color: #4386bf;
```

Comment:
- [ ] `color: #4386bf ` is repeating in several places. You can create a variable with this color at the top and call it where ever u need to assign that color. One of CSS best practice is to avoid repeating the same rule